__tests__/users/App.test.js

```diff
@@ -27,24 +28,34 @@ describe('App Component', () => {
 
     page.on('request', (interceptedRequest) => {
       const url = interceptedRequest.url();
-      if (url === `${API_BASE_URL}/users/inDiscord`) {
+      if (url === `${API_BASE_URL}/users/search/?role=in_discord`) {
```

Comment:
this line is checking if the URL matches the pattern ${API_BASE_URL}/users/search/?role=in_discord. However, this pattern will only match URLs that contain the exact string in_discord in the query string. This means that URLs that contain other values for the role parameter, such as moderator or admin, will not be matched.

Reply:
Yes Tejas it will only match this specific API and that is the requirement as we are only calling only `${API_BASE_URL}/users/search/?role=in_discord` from frontend in according to the feature, tests for other roles and API functionality I have written in the backend.

Reply:
cool then, i am resolving this and leaving for another reviewer